pkg/controller/deliverable/conditions.go

```diff
@@ -111,7 +111,7 @@ func TemplateStampFailureByObservedGenerationCondition(err error) metav1.Conditi
 		Type:    v1alpha1.DeliverableResourcesSubmitted,
 		Status:  metav1.ConditionFalse,
 		Reason:  v1alpha1.TemplateStampFailureResourcesSubmittedReason,
-		Message: fmt.Sprintf("resource '%s' cannot satisfy observedCompletion without observedGeneration in object status", err.(deliverable.RetrieveOutputError).ResourceName()),
+		Message: fmt.Sprintf("Resource [%s] cannot satisfy observedCompletion without observedGeneration in object status", err.(deliverable.RetrieveOutputError).ResourceName()),
```

Comment:
nit: do we want to always capitalize the message?

```console
$ ag 'Message: '
pkg/controller/deliverable/conditions.go
41:             Message: "deliverable has no labels to match to delivery",
50:             Message: fmt.Sprintf("no delivery found where full selector is satisfied by labels: %+v", labels),
59:             Message: "deliverable may only match a single delivery's selector",
68:             Message: deliveryReadyCondition.Message,
87:             Message: err.Error(),
96:             Message: fmt.Sprintf("Resource '%s' is waiting to read value '%s'", resourceName, expression),
105:            Message: err.Error(),
114:            Message: fmt.Sprintf("Resource [%s] cannot satisfy observedCompletion without observedGeneration in object status", err.(deliverable.RetrieveOutputError).ResourceName()),
123:            Message: fmt.Sprintf("Resource [%s] condition not met: %s", err.(deliverable.RetrieveOutputError).ResourceName(), err.(deliverable.RetrieveOutputError).Err.Error()),
132:            Message: fmt.Sprintf("Resource [%s] failed condition met: %s", err.(deliverable.RetrieveOutputError).ResourceName(), err.(deliverable.RetrieveOutputError).Err.Error()),
141:            Message: err.Error(),
150:            Message: err.Error(),
159:            Message: err.Error(),
168:            Message: err.Error(),
```

Reply:
Every where else we use upper case (minus errors). I'll write an issue to look into this.